requirements/edx/base.txt

```diff
@@ -35,9 +35,10 @@ git+https://github.com/edx/django-rest-framework.git@3c72cb5ee5baebc432894737119
 django==1.8.9
 djangorestframework-jwt==1.7.2
 edx-django-oauth2-provider==0.5.0
+edx-djangorestframework-oauth==1.0.1
```

Comment:
Adding a comment, and adjusting the order seems a better approach than having our own fork.